TwitchLib.Client/Extensions/EventInvocationExtensions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using TwitchLib.Communication.Events;
+
+namespace TwitchLib.Client.Extensions
```

Comment:
maybe you could use "File scope namespace" here (I plan to convert the whole project to "file scope namespace" )

Reply:
I could (even tho I kinda hate it), but I tend to keep PRs minimum as possible. We can do file scope namespaces but in separate PR. I don't want to cause Syzuna more headache, because introducing file scope namespaces will cause havoc in PR as it will affect basically all lines in the file and it's nightmare to search for additional changes.

Reply:
I'm aware of what it causes (my big mistake in communication 😢)
and you're right that we can do it in separate RP but if we use FSN for all new files now, then  Syzuna will have less files to check and will have less headaches